src/components/CountryCard/CountryCard.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import './CountryCard.css';
+
+export default function CountryCard({ name, iso2 }) {
+  const iso = iso2.toLowerCase();
+  return (
+    <div className="country-card">
+      <div className="details">
+        <div className="name">{name}</div>
+      </div>
+      <div className="image">
+        <img src={`https://flagcdn.com/32x24/${iso}.png`} />
+      </div>
+    </div>
```

Comment:
Div soup is something to watch for in the future, as a nested group of Div's can cause difficulty when spotting problems. Using semantic html, such as a p tag or h1 tag, for {name} would create a clean code base.